doc/changes.md

```diff
@@ -1,5 +1,95 @@
 # Changelog
 
+## Release v0.10.0 (2023/09/01)
+
+[![Digital Object Identifier for PyGMT v0.10.0](https://zenodo.org/badge/DOI/10.5281/zenodo.8303186.svg)](https://doi.org/10.5281/zenodo.8303186)
+
+### Highlights
+
+* 🎉 **Tenth minor release of PyGMT** 🎉
+*
+* Two new tutorials and seven new gallery examples
```

Comment:
Suggestions for highlights?

Reply:
Merge the two entries in the "New Features" category and move it to "Highlights"?

Reply:
Ok, have combined the two non-ASCII character PRs into one highlight at 81dfe9dbddcf90cb4a5e06537b3f888780abc485. Any other ones, or this is good enough?

Reply:
Hm, a third highlight would be nice, but so far I don't have one 🙁. So, I am OK with two highlights,